app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  after_action :verify_authorized
+  def index
+    authorize :user, :index?
```

Comment:
You don't need `index?` here - Pundit in controllers would do that for you as this already is `index` method.